src/Compilers/VisualBasic/Test/Emit/PDB/PDBAsyncTests.vb

```diff
@@ -0,0 +1,598 @@
+                    <slot kind="temp"/>
+                    <slot kind="temp"/>
+                    <slot kind="temp"/>
+                    <slot kind="temp"/>
+                    <slot kind="temp"/>
+                </encLocalSlotMap>
+            </customDebugInfo>
+            <sequencePoints>
+                <entry offset="0x0" hidden="true" document="0"/>
+                <entry offset="0x7" hidden="true" document="0"/>
+                <entry offset="0x12" startLine="5" startColumn="5" endLine="5" endColumn="58" document="0"/>
+                <entry offset="0x13" hidden="true" document="0"/>
+                <entry offset="0x1e" startLine="6" startColumn="13" endLine="6" endColumn="29" document="0"/>
+                <entry offset="0x2a" startLine="7" startColumn="13" endLine="7" endColumn="29" document="0"/>
+                <entry offset="0x36" startLine="9" startColumn="13" endLine="9" endColumn="53" document="0"/>
+                <entry offset="0x4d" startLine="11" startColumn="9" endLine="11" endColumn="55" document="0"/>
+                <entry offset="0xdd" startLine="12" startColumn="9" endLine="12" endColumn="21" document="0"/>
+                <entry offset="0xee" startLine="13" startColumn="9" endLine="13" endColumn="21" document="0"/>
+                <entry offset="0xff" startLine="14" startColumn="5" endLine="14" endColumn="17" document="0"/>
+                <entry offset="0x101" hidden="true" document="0"/>
+                <entry offset="0x109" hidden="true" document="0"/>
+                <entry offset="0x126" startLine="14" startColumn="5" endLine="14" endColumn="17" document="0"/>
+                <entry offset="0x130" hidden="true" document="0"/>
+            </sequencePoints>
+            <locals>
+                <local name="$VB$ResumableLocal_$VB$Closure_$0" il_index="0" il_start="0x12" il_end="0x100" attributes="0"/>
+                <local name="$VB$ResumableLocal_a$1" il_index="1" il_start="0x12" il_end="0x100" attributes="0"/>
+            </locals>
+            <scope startOffset="0x0" endOffset="0x13d">
+                <importsforward declaringType="C+_Closure$__1-0" methodName="_Lambda$__0"/>
+                <scope startOffset="0x12" endOffset="0x100">
+                    <local name="$VB$ResumableLocal_$VB$Closure_$0" il_index="0" il_start="0x12" il_end="0x100" attributes="0"/>
+                    <local name="$VB$ResumableLocal_a$1" il_index="1" il_start="0x12" il_end="0x100" attributes="0"/>
+                </scope>
+            </scope>
+            <asyncInfo>
+                <kickoffMethod declaringType="C" methodName="Async_Lambda_Hoisted"/>
+                <await yield="0x91" resume="0xb0" declaringType="C+VB$StateMachine_1_Async_Lambda_Hoisted" methodName="MoveNext"/>
+            </asyncInfo>
+        </method>
+    </methods>
+</symbols>)
+        End Sub
+
+        <Fact(), WorkItem(827337, "DevDiv"), WorkItem(836491, "DevDiv")>
+        Public Sub LocalCapturedAndNotHoisted()
```

Comment:
It's not clear how this test is different from the previous test. Perhaps add a comment.